nipype/algorithms/misc.py

```diff
@@ -245,7 +245,7 @@ def _run_interface(self, runtime):
         else:
             affine = None
 
-        with open(self.inputs.header_file, 'rb') as data_file:
+        with open(self.inputs.data_file, 'rb') as data_file:
```

Comment:
I think @alexsavio is right, here. These two lines [were originally](https://github.com/nipy/nipype/commit/a16b7eb356b7f1560f1327eceb0e422ed12fd9b5#diff-ae6a9061a228a455a45ce95549571550L247):

```Python
data = hdr.data_from_fileobj(open(self.inputs.data_file, 'rb'))
```